bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewBindingConstants.java

```diff
@@ -49,6 +49,11 @@
     public static final String CHANNELTYPE_SCENE_ACTIVATE = "scene-activate";
     public static final String CHANNELTYPE_SCENE_GROUP_ACTIVATE = "scene-group-activate";
 
+    public static final String PROPERTY_SHADE_TYPE = "type";
+    public static final String PROPERTY_SHADE_CAPABILITIES = "capabilities";
+    public static final String PROPERTY_SHADE_JSON_HAS_SECONDARY = "jsonHasSecondary";
```

Comment:
Any reason for having "json" in the name? These properties will be seen by users, so seems kind of internal/technical.

Reply:
Hmm. There are two types of properties resp. capabilities -- namely a) those that are **_declared_** through the shade's `type` and `capabilties` attributes, and b) those that are **_inferred_** from the structure of the JSON `positions` response (e.g. if it returns both a `posKind=1` and a `posKind=2` entry then we can infer that it might be a shade with top down and bottom up capabilities). The background is that I created the known types database via some "inspired guesses", and the code is logging any differences between declared and inferred capabilities to provoke users to give us feedback if any of those guesses are wrong. Nevertheless I do agree with you that the 'json' prefix could be confusing for users, and I would be happy to rename the properties. If properties would have long texts, then they would be something like 'potential support for a secondary rail has been detected' and 'potential support for tilt anywhere functionality has been detected' -- and I can't think of a good short text for either of those. => Do you have any suggestions?

Reply:
I agree that removing the "json" prefix would be good. For new names, what about `secondaryRailDetected` and `tiltAnywhereDetected`?

Reply:
Ok. Done.